docs/getting_started/tutorial02.rst

```diff
@@ -1,14 +1,17 @@
 Your First CodeRed Website: Part 2
```

Comment:
I'm thinking we could improve the names to provide more context. Maybe something like: "Tutorial Part 2: Adding Content"

Thoughts?

Reply:
I updated it to be like Your First CodeRed Website: Part 6 -- Classifiers in CMS